include/SDL3/SDL_gpu.h

```diff
@@ -2836,9 +2839,9 @@ extern SDL_DECLSPEC void SDLCALL SDL_EndGPURenderPass(
  */
 extern SDL_DECLSPEC SDL_GPUComputePass *SDLCALL SDL_BeginGPUComputePass(
     SDL_GPUCommandBuffer *command_buffer,
-    const SDL_GPUStorageTextureWriteOnlyBinding *storage_texture_bindings,
+    const SDL_GPUStorageTextureReadWriteBinding *storage_texture_bindings,
     Uint32 num_storage_texture_bindings,
-    const SDL_GPUStorageBufferWriteOnlyBinding *storage_buffer_bindings,
+    const SDL_GPUStorageBufferReadWriteBinding *storage_buffer_bindings,
```

Comment:
Does this need to be explicitly `SDL_GPUStorageBufferReadWriteBinding`? Can it be simplified to `SDL_GPUStorageBufferBinding`?

Reply:
I want it to be clear that writable bindings are the ones that are declared in BeginGPUComputePass.